app/static/js/services/linc.data.factory.js

```diff
@@ -75,6 +75,22 @@ angular.module('linc.data.factory', [])
 		}
 	};
 
+  function refreshOrganizationFilters() {
+    console.log("Performing organization filters refresh...");
+    LincServices.Organizations().then(function (organizations) {
+      var updatedOrgs = _.map(organizations, function(element) {
+        return _.extend({}, element, {checked: true});
+      });
+      options.lions.filters.Organizations = angular.copy(updatedOrgs);
+      options.imagesets.filters.Organizations = angular.copy(updatedOrgs);
+      options.cvrequests.filters.Organizations = angular.copy(updatedOrgs);
+      options.relatives.filters.Organizations = angular.copy(updatedOrgs);
+      $localStorage.options = options;
+    });
+  }
+
+	$rootScope.$on('refreshOrganizations', refreshOrganizationFilters);
```

Comment:
this is the part that does the refresh, using LincServicesOrganizations, which gets all organizations from the db